Oqtane.Client/Services/Interfaces/IPlatform.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Oqtane.UI;
+
+namespace Oqtane.Services
+{
+    public interface IPlatform
+    {
+        Runtime Runtime { get; }
+
+        string Version { get; }
```

Comment:
Version could be used as type `Version` to avoid unnecessary versions comparison